website/docs/r/compute_shared_vpc_service_project.html.markdown

```diff
@@ -37,6 +37,14 @@ The following arguments are expected:
 
 * `service_project` - (Required) The ID of the project that will serve as a Shared VPC service project.
 
+- - -
+
+* `kubernetes_subnetwork_access` - (Optional) A list of strings.  Each string is the self link (or partial self-link)
```

Comment:
It probably makes sense to add something about this conflicting with the `google_compute_subnetwork_iam_*` resources (likewise in their documentation)

Reply:
@danawillow Does this mean you cannot use google_compute_subnetwork_iam_* resources with this field? I think that would be a major problem and hope we can consider a different approach. Subnet IAM applies to far more than just Kubernetes, so it seems like this would prevent users from sharing subnets to Kubernetes and GCE (for example).

Reply:
`subnetwork_iam_member` would be fine - `iam_binding` would conflict.  Do you want me to discard the PR?  I don't have any ideas for another approach - the actual API is just IAM bindings, so anything that claims to be authoritative over subnetwork IAM bindings _will_ conflict, no way around it.

Reply:
Ping @morgante.

Reply:
@ndmckinley Up to you. Personally, I'm not sure this needs to be in the provider. We don't handle normal subnet sharing in the provider either.